settings.yml

```diff
@@ -1,20 +1,21 @@
+# This file relates to internal XMOS infrastructure and should be ignored by external users
 ---
 lib_name: lib_mic_array
 project: '{{lib_name}}'
-title: '{{lib_name}}: PDM Microphone Arrays'
-version: 5.4.0
+title: '{{lib_name}}: PDM Mic Array library for the XMOS XS3 architecture'
```

Comment:
This doesnt match the Readme title

Reply:
Should be "PDM microphone array library"

Reply:
I've changed the title to PDM microphone array library

Reply:
Yeah, test needs extending I guess. For new libs id like to make lib_template or similar.